docs/installation.md

```diff
@@ -307,18 +306,6 @@ faster when formatting a single file from the CLI. The downside to Nailgun is
 that the setup is complicated and the long-running server needs to be restarted
 once in awhile.
 
-### Homebrew
```

Comment:
I think we should keep this header and recommend these steps instead

```sh
brew install coursier/formulas/coursier
cs install scalafmt
cs update scalafmt
```

Reply:
i was thinking that we should refer to coursier's install page (in case something changes in the future) instead of explicit `brew install ...`.

using coursier to install scalafmt is described in its own section above, so wouldn't it be repetitive? (also, i couldn't find `cs` on my mac, with 2.0.0-RC3-4, not until after upgrading to 2.0.0-RC6-15; 2.0.0-RC3-4 also didn't support `install`, said it was experimental.)